data/tabular/bicerano_dataset/meta.yaml

```diff
@@ -0,0 +1,99 @@
+---
+name: polymer_Tg_data
+description: |-
+    This paper outlines a MD simulation workflow based on GPU MD simulation and the
+    refined optimized potentials for liquid simulation (OPLS) OPLS3e force field to
+    calculate glass transition temperatures (Tgs) of 315 polymers for which Bicerano 
+    reported experimental values.
+targets:
+    - id: Tg_exp
+      description: experimental glass transition T
+      units: K
+      type: float
+      names:
+          - noun: glass transition temperature - exp 

```

Comment:
this is a text that is supposed to be part of the language model prompt, e.g. `what is the {noun} of {polymer}`. The `-exp` is a bit strange in this case.